migrations/sql/V2021.04.06.09.53__update_document_templates.sql

```diff
@@ -1,3 +1,18 @@
+INSERT INTO document_template
```

Comment:
Changes to this migration from April 6 should be reverted? I think this is one of the changes you were going to make already.

Reply:
yeah I 100% coppied that migration and then modified the original and left the copy... I will fix that